projects/js-themes-toolkit/packages/liferay-theme-tasks/lib/test/fixtures/themes/7.2/base-theme-7-2/package.json

```diff
@@ -36,7 +36,7 @@
 		"liferay-theme-tasks": "^9.5.0",
 		"compass-mixins": "0.12.10",
 		"liferay-frontend-common-css": "^1.0.4",
-		"liferay-frontend-theme-styled": "^4.0.0-alpha.1552930087997",
-		"liferay-frontend-theme-unstyled": "^4.0.0-alpha.1552930030671"
+		"liferay-frontend-theme-styled": "^6.0.1-test.1c962d5e12be",
+		"liferay-frontend-theme-unstyled": "^6.0.1-test.1c962d5e12be"
```

Comment:
I don't think we want to change this one, right? (I was sure we commented on this in some previous iteration somewhere, but I can't find the comment right now in GitHub or in Gmail...)

When you make a theme for 7.2, you're supposed to use v4 of the themes. Note that this change doesn't affect the lockfile, and it shouldn't affect the tests either.

Reply:
Since it doesn't matter either way for the tests since they reference the root version, is your thought that its just best to keep it the old version for people who might look through these tests?

I'm not opposed to keeping it at the older version, just curious of your thought on it.

Reply:
> Since it doesn't matter either way for the tests since they reference the root version, is your thought that its just best to keep it the old version for people who might look through these tests?

Yep. We have too much confusion around version numbers and dependencies as it is. So if the "truth" is that we expect a specific version of the theme to be used with a specific version of DXP, then we should make sure our tests don't contradict that.